### ISSUE TITLE
[NBug] ConEmuCD was not loaded.

### ISSUE THREAD

**Issue by thomasw-mitutoyo-ctl** [gitextensions/gitextensions#9079]
## Current behaviour

Crashes with callstack and asks to submit this report.

## Expected behaviour

No crash.

## Steps to reproduce

I wanted to commit and push a change in one go.

## Error Details
```
ConEmu.WinForms.GuiMacroExecutor+GuiMacroException: ConEmuCD was not loaded.
   bei ConEmu.WinForms.GuiMacroExecutor.ExecuteInProcessAsync(Int32 nConEmuPid, String asMacro)
   bei ConEmu.WinForms.ConEmuSession.ExecuteGuiMacroTextAsync(String macrotext)
   bei ConEmu.WinForms.GuiMacroBuilder.ExecuteAsync()
   bei ConEmu.WinForms.ConEmuSession.WriteOutputTextAsync(String text)
   bei GitUI.FormStatus.Done(Boolean isSuccess)
   bei GitUI.FormProcess.<>c__DisplayClass43_0.<OnExit>b__0()
   bei GitUI.GitUIExtensions.<InvokeAsync>d__11.MoveNext()
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   bei GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Environment
- Git Extensions 3.2.1.6628
- Build bc07663c3c893f1d896520e0bdcfbe35cfe4b729
- Git 2.31.0.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET Framework 4.8.4341.0
- DPI 96dpi (no scaling)

**Comment by pmiossec** [comment]
I think that has been fixed.
Please update.

**Comment by gerhardol** [comment]
Duplicate of #6675

Corrected in 3.3
